CHANGELOG.md

```diff
@@ -30,10 +30,20 @@
   union other_employees
   ```
 
-- Any other `feat:` commits?
+The following need updated pages in the documentation:
```

Comment:
OK cool, let's ensure we do these before we release; I'll get started on some of them